### DISCUSSION TITLE
Feature request: disable AdNauseam protections for this tab

### DISCUSSION THREAD

**Discussion by Mikaela** [dhowe/AdNauseam#2523] [Ideas]
NoScript has a feature to allow scripts temporarily for anything opened a specific tab intended for doing complicated flows and today I have encountered those twice together with #2522.

- https://noscript.net/usage/#bulkdisabling-restrictions

First I was invited to Microsoft Teams meeting and logging in to Microsoft seems to take 5 or so different domains with some of them also complaining of missing JavaScript support or not just accepting refreshing.

Another case is buying something with [3-D secure](https://en.wikipedia.org/wiki/3-D_Secure) which will require allowing on the webshop, on their payment processor, on your bank website and maybe the bank again has a different subdomain for identification.

Third case is authenticating yourself in Finnish govermental services which first take allowing the service itself and then tunnistus.suomi.fi, maybe yet another domain and unless you are using identity card, your bank comes in too, or in case of mobile ID some subdomain of Telia.

**Comment by mneunomne** [comment]
@Mikaela this is possible to implement yes. It could be a new option in the disable popup "On this tab".

**Comment by dhowe** [comment]
I'm a bit worried about offering the user too many choices in that menu, but its worth a quick mockup
